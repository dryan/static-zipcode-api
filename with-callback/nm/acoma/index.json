{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["87034"], "name": "Acoma", "counties": [{"fips": "061", "name": "Valencia"}]}