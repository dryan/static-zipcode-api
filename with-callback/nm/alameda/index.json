{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["87184", "87114"], "name": "Alameda", "counties": [{"fips": "001", "name": "Bernalillo"}]}