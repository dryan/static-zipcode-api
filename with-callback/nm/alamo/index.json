{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["87825"], "name": "Alamo", "counties": [{"fips": "053", "name": "Socorro"}]}