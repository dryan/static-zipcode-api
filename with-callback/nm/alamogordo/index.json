{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["88311", "88310"], "name": "Alamogordo", "counties": [{"fips": "035", "name": "Otero"}]}