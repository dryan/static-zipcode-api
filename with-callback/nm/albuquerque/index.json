{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["87181", "87185", "87184", "87187", "87125", "87124", "87123", "87122", "87121", "87120", "87109", "87108", "87165", "87101", "87103", "87102", "87105", "87104", "87107", "87106", "87158", "87192", "87193", "87190", "87191", "87196", "87197", "87194", "87195", "87198", "87199", "87131", "87154", "87119", "87153", "87151", "87174", "87113", "87176", "87111", "87116", "87117", "87114", "87115", "87112", "87110"], "name": "Albuquerque", "counties": [{"fips": "001", "name": "Bernalillo"}, {"fips": "043", "name": "Sandoval"}]}