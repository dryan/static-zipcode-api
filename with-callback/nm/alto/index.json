{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["88312"], "name": "Alto", "counties": [{"fips": "027", "name": "Lincoln"}]}