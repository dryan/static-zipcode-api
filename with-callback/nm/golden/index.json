{"region": {"fips": "35", "name": "New Mexico", "abbr": "NM"}, "postal_codes": ["87047"], "name": "Golden", "counties": [{"fips": "001", "name": "Bernalillo"}, {"fips": "049", "name": "Santa Fe"}]}